devops/docker/almalinux-9-amd64/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# TODO: Add MSFT mirror
```

Comment:
Sorry, what's this? We should not have "MSFT" in here. Applies everywhere else.

Reply:
sure, ill take it out